Code/Framework/AzCore/AzCore/RTTI/ReflectContext.h

```diff
@@ -291,6 +451,21 @@ namespace AZ
                 return m_callable;
             }
         }
+
+        virtual bool IsInvokable() const
+        {
+            return AZStd::function_traits<Invocable>::value;
```

Comment:
These is a function for this already.
It is [AZStd::is_invocable](https://en.cppreference.com/w/cpp/types/is_invocable)

That being said this seems to be pushing DOM functionality into the Reflection Context where I don't believe it should belong

Reply:
Also these function shouldn't be virtual. A type being invocable or not is a known at compile time and shouldn't be overridden by a derived.
If anything the this function should `constexpr`